private-eye.js

```diff
@@ -2,21 +2,43 @@
 (function() {
   'use strict';
 
-  var DEFAULT_MESSAGE = "This is a link to a private site, which may or may not be accessible to you.";
+  var DEFAULT_MESSAGE = 'This is a link to a private site, which may or may not be accessible to you.';
+  var STYLES = 'a.private-link::after { content: "\\1F512"; font-size: 0.75em; vertical-align: top; }';
+  var STYLES_ID = '_privateEye-styles';
 
-  var PrivateEye = function(opts) {
+  function setStyles() {
     var styles = document.createElement('style');
-    styles.innerHTML = 'a.private-link::after { content: "\\1F512"; font-size: 0.75em; vertical-align: top; }';
+    styles.innerHTML = STYLES;
+    styles.id = STYLES_ID;
     document.body.appendChild(styles);
+  }
+
+  var PrivateEye = function(opts) {
+    if (!(this instanceof PrivateEye)) {
```

Comment:
Mind adding a comment about why this is here? "Backwards-compatible API support: using `PrivateEye` as a function rather than a class" or something.